doc/manual/interfaces.rst

```diff
@@ -164,9 +164,8 @@ Methods to implement
 :func:`similar(A) <similar>`                                                 ``similar(A, eltype(A), size(A))``           Return a mutable array with the same shape and element type
 :func:`similar(A, ::Type{S}) <similar>`                                      ``similar(A, S, size(A))``                   Return a mutable array with the same shape and the specified element type
 :func:`similar(A, dims::NTuple{Int}) <similar>`                              ``similar(A, eltype(A), dims)``              Return a mutable array with the same element type and the specified dimensions
-:func:`similar(A, ::Type{S}, dims::NTuple{Int}) <similar>`                   ``Array(S, dims)``                           Return a mutable array with the specified element type and dimensions
+:func:`similar(A, ::Type{S}, dims::NTuple{Int}) <similar>`                   ``Array{S}(dims)``                           Return a mutable array with the specified element type and dimensions
 ============================================================================ ============================================ =======================================================================================
-
```

Comment:
sphinx might not like deleting this line but we can deal with that later if it causes any issues